rkt/flag/secflags.go

```diff
@@ -23,21 +23,44 @@ const (
 	insecureOnDisk
 	insecureHTTP
 	insecurePubKey
+	insecureCapabilities
+	insecurePaths
+	insecureSeccomp
 
-	insecureAll = (insecureImage | insecureTLS | insecureOnDisk | insecureHTTP | insecurePubKey)
+	insecureAllFetch = (insecureImage | insecureTLS | insecureOnDisk | insecureHTTP | insecurePubKey)
```

Comment:
I'd consider `insecureOnDIsk` as a part of `insecureAllRun` since it is a runtime option, nothing to do with the fetch.